### DISCUSSION TITLE
Welcome to across Discussions!

### DISCUSSION THREAD

**Discussion by teddysun** [teddysun/across#80] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by girishpatoliya** [comment]
how can I enable ipv6cp in docker ?
I updated /etc/ppp/options.xl2tpd in l2tp docker instance and committed it, but on new instance it got overwritten.

**Comment by xianrennet** [comment]
您好，您的邮件已收到，我将尽快回复您。如是删除信息，我们将在5个工作日内处理，请尽量使用密码删除信息，如预留邮箱，可自助找回密码，不便之处敬请谅解。------------------------------本邮件为自动回复，谢谢！
